src/azul/indexer/mirror_service.py

```diff
@@ -177,12 +177,9 @@ def _get_info(self, catalog: CatalogName, file: File) -> JSON | None:
             return None
         else:
             json_content = json.loads(content)
-            # FIXME: Convert to a warning
-            #        https://github.com/DataBiosphere/azul/issues/7130
-            if False:
-                content_type = json_content['content-type']
-                assert content_type == file.content_type, R(
-                    'Conflicting content type', content_type, file)
```

Comment:
We need a new FIXME here (separate commit).

https://github.com/DataBiosphere/azul/issues/7193